pxtarget.json

```diff
@@ -551,6 +551,15 @@
         "python": true,
         "assetEditor": true
     },
+    "queryVariants": {
+        "skillsMap=1": {
```

Comment:
should we make this singular? not blocking for this pr

Reply:
yeah probably! i'll need to update pxt-core too